cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/media/class-video.php

```diff
@@ -348,7 +348,6 @@ public function print_video_scripts() {
 				$default       = array(
 					'publicId'    => $cloudinary_id,
 					'sourceTypes' => array( $video['format'] ), // @todo Make this based on eager items as mentioned above.
-					'controls'    => 'on' === $this->config['video_controls'] ? true : false,
```

Comment:
@dugajean 

Do we want to remove the `controls` key from the default array?

Reply:
It's being added below under all circumstances, so no need for duplicate code :)